pages/tools/slack/emoji.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Slack emoji usage and best practices
+questions:
+  - admins-slack
+cSpell: 
+---
+
+## Custom emoji
+
+Emoji are an important part of our communication within Slack. You are welcome
+to add custom emoji. Keep in mind that interactions in Slack are covered by our
+[Code of Conduct]({% page "/about-us/code-of-conduct/" %}), including emoji.
+Please be thoughtful and mindful about emoji submissions and ensure they're in
+line with our values of diversity, equity and inclusion.
+
+Before requesting emoji featuring TTS members' faces, family, pets, etc., please
+**request consent** from those featured. Emoji reviewers are required to check
+that consent was obtained.
+
+All emoji must be 128KB or smaller with a caveat: sometimes Slack may reject
```

Comment:
Add formats?

Reply:
Can you share the format requirements? I don't know them.